bridetalk_project/src/components/pages/masterlayout.css

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Since MasterLayout is a component, why don't u have a MasterLayout folder that has the component and its CSS? It would be also nice if you could give the CSS file the same name as the component.

`masterlayout.css => MasterLayout.css`